include/lbann/data_readers/data_reader_imagenet.hpp

```diff
@@ -31,6 +31,7 @@
 
 #include "data_reader_image.hpp"
 #include "cv_process.hpp"
+#include "opencv_extensions.hpp"
```

Comment:
This might not be necessary although it does not hurt anything. cv_process.hpp already includes cv_transform.hpp which includes opencv_extensions.hpp